File: farondroid/src/main/java/com/openfarmanager/android/view/ToastNotification.java
package com.openfarmanager.android.view;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;
import com.openfarmanager.android.App;
import com.openfarmanager.android.R;

public class ToastNotification {

    private static View sToastView;

    static {
        sToastView = LayoutInflater.from(App.sInstance.getApplicationContext()).inflate(R.layout.toast_notification_view, null);
    }

    public static Toast makeText(Context context, CharSequence text, int duration) {
        Toast toast = new Toast(context);
        toast.setDuration(duration);
        //toast.setDuration(Toast.LENGTH_LONG);

        ((TextView) sToastView.findViewById(R.id.text)).setText(text);

        toast.setView(sToastView);
        return toast;
    }

}

File: farv2/src/main/java/com/openfarmanager/android/filesystempanel/view/ToastNotification.kt
package com.openfarmanager.android.filesystempanel.view

import android.content.Context
import android.view.LayoutInflater
import android.widget.Toast
import com.openfarmanager.android.R
import kotlinx.android.synthetic.main.toast_notification_view.view.*

class ToastNotification {

    fun makeText(context: Context, text: CharSequence, duration: Int): Toast {
        val view = LayoutInflater.from(context).inflate(R.layout.toast_notification_view, null)
        val toast = Toast(context)
        toast.duration = duration
        //toast.setDuration(Toast.LENGTH_LONG);
        view.text.text = text
        toast.view = view
        return toast
    }
}